benchmark/realnest/hep/hep.benchmark.in

```diff
@@ -0,0 +1,13 @@
+# name: ${FILE_PATH}
+# description: ${DESCRIPTION}
+# group: [hep]
+
+require httpfs
+
+cache hep_singleMu.duckdb
```

Comment:
can you also add a comment somewhere here with a link to the benchmarks. Thanks!